packages/components/package.json

```diff
@@ -66,6 +66,7 @@
     "@giscus/react": "3.0.0",
     "@next/bundle-analyzer": "13.4.2",
     "clsx": "2.1.0",
+    "date-fns": "3.6.0",
```

Comment:
maybe this could be a ranged dependency? something like `^3.6.0`? otherwise it might be annoying for other apps to use.